core/vm/actor/init_actor.cpp

```diff
@@ -19,4 +20,35 @@ namespace fc::vm::actor {
     ++next_id;
     return Address::makeFromId(id);
   }
+
+  outcome::result<InvocationOutput> InitActor::exec(
+      const Actor &actor, Runtime &runtime, const MethodParams &params) {
+    OUTCOME_TRY(exec_params, decodeActorParams<ExecParams>(params));
+    if (!isBuiltinActor(exec_params.code)) {
+      return NOT_BUILTIN_ACTOR;
```

Comment:
error enums must be part defined as `enum class SomeErrorEnum {...}`, possibly as integers: 
`enum class SomeErrorEnum: som_integer_type {}`. In this case method would return `SomeErrorEnum::NOT_BUILTIN_ACTOR`; `SomeErrorEnum::SINGLETON_ACTOR` and so on.

Reply:
I see, it is due to VMExitCode. Anyway, pls consider refactoring allowing to use enums instead of VMExitCode.

Reply:
At least pls use full name. I mean `InitActor:: NOT_BUILTIN_ACTOR` and so on.